src/activities/Elsa.Activities.RabbitMq/Services/Client.cs

```diff
@@ -0,0 +1,90 @@
+using Elsa.Activities.RabbitMq.Configuration;
+using Elsa.Activities.RabbitMq.Decorators;
+using Rebus.Activation;
+using Rebus.Bus;
+using Rebus.Config;
+using Rebus.Messages;
+using Rebus.Routing.TransportMessages;
+using Rebus.Transport;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Elsa.Activities.RabbitMq.Services
+{
+    public class Client : IClient
+    {
+        private BuiltinHandlerActivator _activator { get; set; }
+        private IBus _bus { get; set; }
```

Comment:
Properties should be named `PascalCased`. Alternatively, these private members can be turned into fields, in which case the casing is correct as it is.